cpt-gateway-services.rst

```diff
@@ -36,22 +36,19 @@ Release manager (RM)
   to clients.
 
 Repository Gateway Configuration
-==============================
+================================
 
 As a prerequisite, we need to install the CVMFS client and server
 packages on the gateway. This means that the gateway machine can be
 used as a "master" release manager to perform some repository
 transformations before a separate release manager machine
 is set up.
 
-The repository gateway application is packaged as a tarball, currently available for Ubuntu 16.04, SLC 6 and Cern CentOS 7. The tarball should be unpacked into ``/opt/cvmfs_services``: ::
+The repository gateway application is currently packaged for Ubuntu
+16.04, SLC 6 and Cern CentOS 7. Once the package is installed, the
+setup script needs to be run: ::
 
-  $ cd /opt/cvmfs_services
-  $ tar xzf cvmfs_services-0.1.10-cc7-x86_64.tar.gz
-
-Then, run the set up script: ::
-
-  $ /opt/cvmfs_services/scripts/setup.sh
+  $ /opt/cvmfs_gateway/scripts/setup.sh
```

Comment:
I hope the plan is to move this out of /opt, because that's not the normal place for native-packaged software.